File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
       for(int i = 0;i<nums.length;i++){
        for(int j=i+1;j<nums.length;j++){
            //check if the pair sums to the target
            if(nums[i] + nums[j]== target){
                //return the indices of the valid pair
                return new int[] {i,j};
            }
         }
       } 
       // return [-1,-1] if there is no pair found to meet the target value
       return new int[] {-1,-1};
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        #create dictionary to store
        num_dict = {}
        for i, num in enumerate(nums):
            complement = target - num
            #if complement exists in dictionary, return both indices
            if complement in num_dict:
                return[num_dict[complement],i]
            num_dict[num] = i
        return []
